utils/generate-domains-blacklists/domains-blacklist.conf

```diff
@@ -161,3 +161,6 @@ https://dbl.oisd.nl/
 
 # Block spying and tracking on Windows 
 # https://raw.githubusercontent.com/crazy-max/WindowsSpyBlocker/master/data/dnscrypt/spy.txt
+
+# GameIndustry.eu - Blockieren Sie Spyware, Advertising, Analytics, dem Tracking in Spielen und in dazugehörigen Clients.
```

Comment:
Does the description need to be in German? :)

Reply:
I'll use DeepL to translate it into English.